src/django_components/provide.py

```diff
@@ -28,10 +27,7 @@ def __init__(
     ):
         super().__init__(nodelist=nodelist, args=None, kwargs=kwargs, node_id=node_id)
 
-        self.nodelist = nodelist
-        self.node_id = node_id or gen_id()
         self.trace_id = trace_id
-        self.kwargs = kwargs or RuntimeKwargs({})
```

Comment:
Not related to the PR, just noticed that this is extra, as the same assignment happens in `super().__init__`